tests/end2end/features/downloads.feature

```diff
@@ -337,6 +337,20 @@ Feature: Downloading things from a website.
         And I open the download with a placeholder
         Then "Opening *download.bin* with [*python*]" should be logged
 
+    Scenario: Opening a download with default-open-dispatcher set
+        When I set general -> default-open-dispatcher to python {} -c "import sys; print(sys.argv[1])"
```

Comment:
Is this just so it uses the correct version of python for testing? Because it practice you shouldn't have to enter the full path to executables.